app/api/v1/helpers/crud_helpers.rb

```diff
@@ -0,0 +1,13 @@
+module V1
+  module Helpers
+    module CrudHelpers
+      def sorting(class_name, sort_by, default_sort)
+        if sort_by[:column_name] && sort_by[:type]
+          class_name.order("#{sort_by[:column_name]} #{sort_by[:type]}")
+        else
+          class_name.order("#{default_sort[:column_name]} #{default_sort[:type]}")
```

Comment:
your default sort can be just asc by id for all collections
in this case you don't have to pass `default_sort` to this method in every controller